Project/src/MakeCall/StreamRenderer.js

```diff
@@ -156,7 +175,9 @@ export default class StreamRenderer extends React.Component {
         if (this.renderer) {
             this.renderer.dispose();
```

Comment:
Can you please check if the html div is being removed correctly from the dom?

Reply:
This renderer.dispose() removed the div that the sdk gives to us. But i think we're not cleaning up this component's div correctly.

Reply:
Yes, that is correct, the html keeps there. I was trying to solve that as well but weren't able to figure that out yet, I will keep investigating and add that in another PR